arm_isa.cpp

```diff
@@ -1377,7 +1377,7 @@ inline void LDRH(arm_isa* ref, int rd, int rn,
     printf("Unpredictable LDRH instruction result (Address is not Halfword Aligned)\n");
     return;
   }
-  value = MEM.read(ref->ls_address.entire);
+  value = MEM.read_half(ref->ls_address.entire);
   value &= 0xFFFF; /* Zero extends halfword value 
 		      BUG: Model must be little endian in order to the code work  */
```

Comment:
Ok. I removed this line specifically. But the arm_isa.cpp has others 0xFFFF mask  that freezes the model in little endian only.